Sources/Rendering/Misc/FullScreenRenderWindow/index.js

```diff
@@ -188,19 +185,25 @@ function vtkFullScreenRenderWindow(publicAPI, model) {
 // Object factory
 // ----------------------------------------------------------------------------
 
-const DEFAULT_VALUES = {
-  background: [0.32, 0.34, 0.43],
-  containerStyle: null,
-  controlPanelStyle: null,
-  listenWindowResize: true,
-  resizeCallback: null,
-  controllerVisibility: true,
-};
+function defaultValues(initialValues) {
+  return {
+    background: [0.32, 0.34, 0.43],
+    containerStyle: null,
+    controlPanelStyle: null,
+    listenWindowResize: true,
+    resizeCallback: null,
+    controllerVisibility: true,
+    ...initialValues,
+  };
+}
 
 // ----------------------------------------------------------------------------
 
 export function extend(publicAPI, model, initialValues = {}) {
-  Object.assign(model, DEFAULT_VALUES, initialValues);
+  // a background property can be set in the initialValues but we do not want
+  // to "store" background, only forward it to the renderer
```

Comment:
remove comment

Reply:
Maybye it could be better to have that information in the index.d.ts file or somewhere, in order that users know that when they pass it in the initialValues it will not be in the model afterwards.

Reply:
ok to document 'background' property in index.d.ts for `newInstance()`. No need to explain it is not "stored", this is an implementation detail, it is not a useful information for the "user".